src/realm/object-store/c_api/sync.cpp

```diff
@@ -47,13 +47,17 @@ realm_app_user_subscription_token::~realm_app_user_subscription_token()
 
 namespace realm::c_api {
 
+#if REALM_APP_SERVICES
+
 static_assert(realm_sync_client_metadata_mode_e(app::AppConfig::MetadataMode::NoEncryption) ==
               RLM_SYNC_CLIENT_METADATA_MODE_PLAINTEXT);
 static_assert(realm_sync_client_metadata_mode_e(app::AppConfig::MetadataMode::Encryption) ==
               RLM_SYNC_CLIENT_METADATA_MODE_ENCRYPTED);
 static_assert(realm_sync_client_metadata_mode_e(app::AppConfig::MetadataMode::InMemory) ==
               RLM_SYNC_CLIENT_METADATA_MODE_DISABLED);
 
+#endif // REALM_APP_SERVICES
+
```

Comment:
Should we also move these to `c_api/app.cpp` as well? It seems to be the only "App-y" thing left in this file...

Reply:
Good idea - done.